### DISCUSSION TITLE
Template for ticket

### DISCUSSION THREAD

**Discussion by Stefke232** [arnoudkooi/SN-Utils#310] [General]
Is there a way to create / command that will fill some parts of the ticket? Like a template that a can use for repetitive tasks in tickets.

**Comment by arnoudkooi** [comment]
Can you check if [this video](https://www.youtube.com/watch?v=7IIPv8DO6jY&t=8s&ab_channel=ArnoudKooi) helps?

**Reply by Stefke232** [reply]
This is very nice option but how do I make it work on existing ticket.
For example I want to make ticket resolution automated and to apply template with / command.

**Reply by tdeniston** [reply]
You can have the command state "javascript:applyTemplate('[sys_id of the template]')". This will cause it to apply the template on the client-side. 

![image](https://user-images.githubusercontent.com/15730875/194175755-7e6afd7e-ec61-43de-9bf3-ad878ba322f4.png)

**Reply by arnoudkooi** [reply]
Great tip @tdeniston, thanks. Hope that works @Stefke232!
In an old review of the Safari version I just now see you mentioned high CPU usage. Is that still the case? (Hope not, as I moved away from the persistent background page)

**Reply by tdeniston** [reply]
I don't use Safari much, as I have a much beefier computer now that can handle dozens of tabs in Chrome. 😄 
I just fired it up and reinstalled the latest version of SN Utils for Safari. Seems to no longer have the CPU issue.

**Reply by arnoudkooi** [reply]
Thanks, appreciate you checking!

**Reply by Stefke232** [reply]
Hi,
I have tried this idea to apply existing template but it is not working.
Running the command just creates new empty tab and there are no changes in the ticket.
